src/Core/Math/Math.inl

```diff
@@ -2,6 +2,12 @@
 
 namespace Ra {
 namespace Core {
+
+template <typename T>
+inline constexpr T clamp( T v, T min, T max ) {
+    return std::max( min, std::min( v, max ) );
+}
+

```

Comment:
I guess this function can go away if we use `std::clamp` instead

Reply:
well, yes, I was just lazy ;) but it will be done soon